NAMESPACE

```diff
@@ -63,7 +63,10 @@ export(
        "BinaryOption",
        "BarrierOption",
        ##--schedule.R
-       "Schedule"
+       "Schedule",
+       ##--FXDerivatives.R
+       "oneTouch",
+       "DNTVannaVolga"
```

Comment:
I would like to see see names more in lines with the current naming schemes.   Can you suggest alternatives?